packages/ecr-credential-provider-1.29/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+# "." is not allowed in crate names, but we want a friendlier name for the
+# directory and spec file, so we override it below.
+name = "ecr-credential-provider-1_27"
```

Comment:
This is confusing. This should be 1.29 not 1.27? Same with the spec file naming.

Reply:
yeah, it's typo. I'll fix it and re-test it.  thanks!